packages/react/src/utils/styles.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+
+export const STYLE_DISABLE_SCROLLBAR = {
+  className: 'base-ui-disable-scrollbar',
+  element: (
+    <style href="/base-ui-disable-scrollbar">
```

Comment:
It should also have the precedence attribute for React 19 to hoist it to `head`.

Reply:
What's the starting `/` for in `href` ?

Reply:
Fixed, thanks